front/components/agent_builder/AgentBuilderPerformance.tsx

```diff
@@ -0,0 +1,214 @@
+import {
+  Avatar,
+  Button,
+  CardGrid,
+  ChatBubbleLeftRightIcon,
+  ChatBubbleThoughtIcon,
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+  HandThumbDownIcon,
+  HandThumbUpIcon,
+  Page,
+  Spinner,
+  ValueCard,
+} from "@dust-tt/sparkle";
+import { useState } from "react";
+
+import { useAgentBuilderContext } from "@app/components/agent_builder/AgentBuilderContext";
+import { FeedbacksSection } from "@app/components/assistant_builder/FeedbacksSection";
+import { useAgentAnalytics } from "@app/lib/swr/assistants";
+import type { AgentConfigurationType } from "@app/types";
+import { removeNulls } from "@app/types";
+
+const PERIODS = [
+  { value: 7, label: "Last 7 days" },
+  { value: 15, label: "Last 15 days" },
+  { value: 30, label: "Last 30 days" },
+];
+
+const DEFAULT_PERIOD_VALUE = 30;
```

Comment:
very nit: make sure we cannot shoot ourselves in the foot
```suggestion
const PERIODS = [
  { value: 7, label: "Last 7 days" },
  { value: 15, label: "Last 15 days" },
  { value: 30, label: "Last 30 days" },
] as const;

type PeriodValue = (typeof PERIODS)[number]["value"];
const DEFAULT_PERIOD_VALUE: PeriodValue = 30;
```
Also it automatically makes the code using below safer.
```
// Period can only be 7 / 15 / 30
const [period, setPeriod] = useState(DEFAULT_PERIOD_VALUE);
```